### PR TITLE
Dice coefficient

### PR DESCRIPTION
**Pull request by muskankhedia**
Fixes: #65
Added the dice_coefficient.go file in a modular way for comparison of a string with a set of strings and return the rating for the word according to the comparison in order to choose the best possible case.
Screenshot at 2019-04-07 01-38-03

### CODE REVIEW

File: service/controllers/dice_coefficient.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+			intersectionSize++
+		}
+	}
+
+	var result int
+	result = (2.0 * intersectionSize * 100)  / (len(first) + len(second) - 2)
+	return (result)
+}
+
+//FindBestMatch ...
```
Comment (Harkishen-Singh) [inline service/controllers/dice_coefficient.go]: write a better description

**Comment by Harkishen-Singh** [comment]
could you please attach a test video, testing on a sample of strings?

File: service/controllers/dice_coefficient.go (hunk 1 above)
Comment (muskankhedia) [reply service/controllers/dice_coefficient.go]: Yaa, adding it in the next commit. Thank you!

**Comment by muskankhedia** [comment]
Yes Sir, adding in some time.

**Pull request merged**